### ISSUE TITLE
Warning: NativeResourceHolder - Should have been closed before finalization

### ISSUE THREAD

**Issue by BilliAlpha** [sedmelluq/lavaplayer#114]
Hello,
It seems that the RessourceHolder isn't closed after the stream is finished and is automatically closed on object destrcution.
I've seen this warning when an ogg (Vorbis) track finished playing.
It's just a warning so nothing really important but I wanted to make sure that if this was a bug you knew about it.

**Comment by sedmelluq** [comment]
Can you reproduce it every time with some specific track? Are there any exceptions in the log before that? Also please show the full warning message, it should give the specific resource class name.

The resources are indeed supposed to be all closed normally, although exceptions thrown in some places may cause some close methods to not be called.

**Comment by BilliAlpha** [comment]
Hi,
The warning appears every time I play any track,
```
2018-05-13 18:55:29 WARN  com.sedmelluq.discord.lavaplayer.natives.NativeResourceHolder - Should have been closed before finalization (com.sedmelluq.discord.lavaplayer.natives.vorbis.VorbisDecoder).
```
There are no exception and sometimes the warning appears before the song ends, probably linked to something happening during track loading.

**Comment by sedmelluq** [comment]
Fixed as of `1.3.0`.
